api/hub/src/worker.js

```diff
@@ -0,0 +1,5 @@
+const path = require('path');
```

Comment:
you don't need to create this file as .js, just make it .ts and use the lib path for imports

Reply:
I've tried to do that, but passing a  .ts file to a worker thread results in error:
```
The worker script extension must be “.js” or “.mjs”. Received “.ts”
```

This workaround allows us to use .ts files for the actual worker code, which then gets loaded by the .js file.

Reply:
the tsc compilation step would create the .js file for you

Reply:
just make this file .ts, remove ts-node and keep everything else

Reply:
because after the compilation step everything will happen under the lib folder where everything is already .js